tests/composer/composer-symfony6-max.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "propel/propel",
+    "type": "library",
+    "description": "Propel2 is an open-source Object-Relational Mapping (ORM) for PHP 5.5 and up.",
+    "keywords": [
+        "ORM",
+        "persistence",
+        "Active Record"
+    ],
+    "homepage": "http://www.propelorm.org/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "William Durand",
+            "email": "william.durand1@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=8.0",
+        "symfony/yaml": "^6.0.0",
+        "symfony/config": "^6.0.0",
+        "symfony/console": "^6.0.0",
+        "symfony/filesystem": "^6.0.0",
+        "symfony/finder": "^6.0.0",
+        "symfony/translation": "^6.0.0",
+        "symfony/validator": "^6.0.0",
+        "symfony/polyfill-php73": "^1.8",
```

Comment:
Do we still need that polyfill? We use >=8.0.

Reply:
I guess it is easier though to keep the files similar across all versions.
We should remove things everywhere then if possible/useful.

Reply:
I copy/pasted from other min/max composer.json files in this directory.

I will remove it as the polyfill is now absent from main composer.json.